api/v1alpha3/machine_types.go

```diff
@@ -29,6 +29,14 @@ const (
 	// MachineControlPlaneLabelName is the label set on machines or related objects that are part of a control plane.
 	MachineControlPlaneLabelName = "cluster.x-k8s.io/control-plane"
 
+	// ConditionHealthCheckSucceeded is set on machines that have passed a healthcheck by the MachineHealthCheck controller.
+	// In the event that the health check fails it will be set to False.
+	ConditionHealthCheckSucceeded ConditionType = "HealthCheckSucceeded"
+
+	// ConditionOwnerRemediated is set on machines that have failed a healthcheck by the MachineHealthCheck controller.
+	// ConditionOwnerRemediated is set to False after a health check fails, but should be changed to True by the owning controller after remediation succeeds.
+	ConditionOwnerRemediated ConditionType = "OwnerRemediated"
```

Comment:
```suggestion
	MachineOwnerRemediatedCondition ConditionType = "OwnerRemediated"
```

This should also be moved to `condition_consts.go`